drivers/dai/intel/ssp/dai-params-intel-ipc4.h

```diff
@@ -228,12 +231,18 @@ struct dai_intel_ipc4_ssp_config {
 	uint32_t ssc1;
 	uint32_t sscto;
 	uint32_t sspsp;
+#ifndef CONFIG_SOC_INTEL_ACE30_PTL
 	uint32_t sstsa;
 	uint32_t ssrsa;
+#endif
 	uint32_t ssc2;
 	uint32_t sspsp2;
 	uint32_t ssc3;
 	uint32_t ssioc;
+#ifdef CONFIG_SOC_INTEL_ACE30_PTL
+	uint64_t ssmidytsa[I2SIPCMC];
+	uint64_t ssmodytsa[I2SOPCMC];
```

Comment:
Thanks for dropping the `struct`, unfortunately the cryptic name is back and you also dropped the explanation...

Reply:
Will be updated with new PR, thanks